src/net_processing.cpp

```diff
@@ -4396,7 +4396,7 @@ bool PeerLogicValidation::SendMessages(CNode* pto)
             int64_t timeNow = GetTimeMicros();
             if (timeNow > pto->m_tx_relay->nextSendTimeFeeFilter) {
                 static CFeeRate default_feerate(DEFAULT_MIN_RELAY_TX_FEE);
```

Comment:
Can make `default_feerate` const too -- could make it `constexpr` even, so long as the constructors were also marked constexpr.